packages/lesswrong/components/common/Divider.tsx

```diff
@@ -37,17 +43,23 @@ const Divider = ({ classes, wings=true }: {
   classes: ClassesType,
   wings?: boolean
 }) => {
-  return <div className={classes.root}>
-    {wings && <div className={classes.divider}>
-      { divider }
-    </div>}
-    <div className={classes.compass}>
-      { compass }
-    </div>
-    {wings && <div className={classes.divider}>
-      { divider }
-    </div>}
-  </div>
+  return forumTypeSetting.get() === "EAForum"
+    ? (
+      <MuiDivider className={classes.eaDivider} />
+    )
```

Comment:
Good call - I originally thought there was going to be multiple books but it seems like this won't be the case for now. I've gone ahead and removed them.

Reply:
I've also improved the font a little by making it more similar to the main heading on the page and the heading used on the sequences page. There's probably more that can be done here to make it look better, but I think it will be clearer what to do once more of the content is in place. The book title, for instance, is still in a serif font which probably isn't correct, but I'll fix this later. All the subtitles look a bit out-of-place too, but LW doesn't actually use the subtitles so I'll wait to see if we have a use for them or not.
<img width="438" alt="Screenshot 2022-07-11 at 11 35 20" src="https://user-images.githubusercontent.com/5075628/178246081-ca1c9927-db64-4d99-b4b1-f327c20f6d48.png">

The issue with the word count seems to just be that the posts I added in the dev database were really old and the wordcount is null for some reason. I just edited the post to recalculate the word count at it seems to be working correctly now:
<img width="171" alt="Screenshot 2022-07-11 at 11 44 58" src="https://user-images.githubusercontent.com/5075628/178247678-49e600a8-7b66-4dd9-9b0b-2926f7c85e02.png">